daisy_workflows/image_build/enterprise_linux/enterprise_linux_uefi.wf.json

```diff
@@ -145,6 +151,39 @@
         }
       ]
     },
+    "export-metadata": {
+      "CreateInstances": [
+        {
+          "Name": "inst-exportmetadata",
+          "Disks": [{"Source": "disk-installerprep"}, {"Source": "disk-installer"}],
+          "MachineType": "n1-standard-4",
+          "Metadata": {
+            "files_gcs_dir": "${SOURCESPATH}/build_files",
+            "prefix": "ExportMetadata",
+            "script": "export_metadata.py",
+            "build_date": "${build_date}",
```

Comment:
i don't think build_date is passed in to this workflow. i don't see it in the vars section. i think it's only available in the workflow which includes this one.

Reply:
Should it be explicitly included in the IncludWorkFlow.Vars?. I thought it's inherit from parent workflow.

Reply:
Using datetime to directly generate it to avoid pass in.